codereview.java

```diff
@@ -0,0 +1,36 @@
+public class RuntimeIssuesDemo {
+
+    public static void main(String[] args) {
+        int[] numbers = {1, 2, 3};
+        System.out.println(numbers[3]); 
+
+        
+        String str = null;
+        System.out.println(str.length()); 
+
+        
+        int result = 10 / 0; 
+
+    
+        String numStr = "abc";
+        int num = Integer.parseInt(numStr);
```

Comment:
<b>code: int num = Integer.parseInt(numStr); - </b>
NumberFormatException might occur if numStr is not a valid integer